frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/ModelManager.java

```diff
@@ -102,6 +114,11 @@ public ModelArchive registerModel(
             throw new BadRequestException("Model " + modelName + " is already registered.");
         }
         logger.info("Model {} loaded.", model.getModelName());
+
+        if (startupModel) {
+            ModelManager.getInstance().setStartupModels(model);
```

Comment:
```suggestion
            startupModels.add(model);
```

Reply:
I don't see a need for model. I redid this PR with just the model-name. It works for most cases and has less references of "model" object itself. This may be better from memory clean up perspective